### ISSUE TITLE
opam var --package is slow on macos

### ISSUE THREAD

**Issue by rgrinberg** [ocaml/opam#5536]
See output [here](https://gist.github.com/rgrinberg/3d9a733b4c737a5f261abb1d9880ed92)

Note that printing individual vars is fast.

**Comment by rjbou** [comment]
What version of opam are you using ?
From logs, it seems that slow part is the tar repo mechanism (disabled in #5015). Can you check with master if you have the same behaviour ?

**Comment by kit-ty-kate** [comment]
I agree with @rjbou's assessment. I'll close this since this was fixed back in opam 2.2 by the PR linked above.
